### ISSUE TITLE
Currently used version of logback-core contains a security vulnerability

### ISSUE THREAD

**Issue by fredsjones** [deeplearning4j/deeplearning4j#4690]
In our exploration of your project we found that it is currently using version 1.1.2 of logback-core which is vulnerable to Arbitrary Code Execution. See the links below for the details on the vulnerability.

Recommendation:

Upgrade the version of logback-core in the pom.xml to version 1.2 or higher.

For additional details on this vulnerability you can visit the following websites:
Snyk: 
https://snyk.io/vuln/SNYK-JAVA-CHQOSLOGBACK-30208

**Comment by raver119** [comment]
Could you please point me to the place where we use that EXCEPT of test scope please?

**Comment by fredsjones** [comment]
You are correct, it is currently only used in the test scope.  Its something to consider updating at some point since the risk is low and all it takes is one person to start using outside the test scope to potentially (not guaranteed) to increase the risk.

**Comment by raver119** [comment]
Ok, thank you.
